kyuubi-server/src/test/scala/org/apache/kyuubi/engine/JpsApplicationOperationSuite.scala

```diff
@@ -81,12 +81,11 @@ class JpsApplicationOperationSuite extends KyuubiFunSuite {
       assert(desc1.contains("id"))
       assert(desc1("name").contains(id))
       assert(desc1("state") === "RUNNING")
+      val response = jps.killApplicationByTag(id)
+      assert(response._1, response._2)
+      assert(response._2 startsWith "Succeeded to terminate:")
     }
 
-    val response = jps.killApplicationByTag(id)
-    assert(response._1, response._2)
-    assert(response._2 startsWith "Succeeded to terminate:")
-
     val desc2 = jps.getApplicationInfoByTag(id)
```

Comment:
Since we kill the application with signal `SIGTERM`, we should surround L90-L93 with `eventually` too.

[SIGTERM vs SIGKILL: What's the Difference?](https://linuxhandbook.com/sigterm-vs-sigkill/)
```
SIGTERM gracefully kills the process whereas SIGKILL kills the process immediately.
```